config/initializers/acts_as_watchable.rb

```diff
@@ -1,8 +1,13 @@
 # Be sure to restart your server when you modify this file.
 
-# In development and non-eager loaded mode, we need to register the acts_as_watchable models manually
+# For development and non-eager load mode, we need to load models using acts_as_watchable manually
 # as no eager loading takes place
-Rails.application.config.after_initialize do
-  OpenProject::Acts::Watchable::Registry
-    .add(WorkPackage, Message, Forum, News, Meeting, Wiki, WikiPage)
+Rails.application.config.to_prepare do
+  Forum
+  Meeting
+  Message
+  News
+  Wiki
+  WikiPage
+  WorkPackage
```

Comment:
I guess this is due to this: 

> Both registries were storing model classes, so in development new versions of those models were added when reloaded, but the registry still returned the first model class that was registered

But this could be fixed in the to_prepare call to simply clear out the registry

Reply:
I too prefer adding it explicitly